### ISSUE TITLE
after tipping, the banner does not have good keyboard accessibility

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#7611]
## Description 
After you tip, the "thank you" message is in-line on the tip banner in 1.2.x and above. However, after tipping there is no element highlighted (showing a box around it) to indicate that it is active on the tip banner. This is necessary for accessibility so users can know where they are on the banner. If you tab a bit eventually something will get highlighted, but then you encounter https://github.com/brave/brave-browser/issues/6986 and leave the tip banner, unable to return via tab.

## Steps to Reproduce

   1. Have rewards enabled and BAT in your wallet.
   2. Go to a verified publisher.
   3. Open tip banner.
   4. Tip site (note, thank you is "in line" in 1.2.x and above)
   5. Thank you message is displayed where the tip amounts were.

## Actual result:
Nothing on the tip banner is highlighted w/ a box to indicate where you are - same occurs for both monthly and one time tips:
<img width="1189" alt="Screen Shot 2020-01-06 at 10 05 04 AM" src="https://user-images.githubusercontent.com/28145373/71837875-73372980-3085-11ea-9086-7b2992aa172f.png">

## Expected result:
Something should be highlighted and user selectable via keyboard - perhaps the Share option? This is for product to determine.

## Reproduces how often: 
easily

## Brave version (brave://version info)

Brave | 1.2.41 Chromium: 79.0.3945.88 (Official Build) (64-bit)
-- | --
Revision | c2a58a36b9411c80829b4b154bfcab97e581f1f3-refs/branch-heads/3945@{#954}
OS | macOS Version 10.14.6 (Build 18G103)

## Version/Channel Information:

- Can you reproduce this issue with the current release? 1.1.x it is n/a, but yes for 1.2.x
- Can you reproduce this issue with the beta channel? probably
- Can you reproduce this issue with the dev channel? probably
- Can you reproduce this issue with the nightly channel? yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?
- Does the issue resolve itself when disabling Brave Rewards?
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:
cc @mandar-brave @alexwykoff

**Comment by emerick** [comment]
Unable to repro. I think the most recent redesign of the tipping banner ended up addressing this, but please reopen if I'm incorrect.

**Comment by LaurenWags** [comment]
Verified with

```
Brave | 1.43.61 Chromium: 104.0.5112.81 (Official Build) beta (x86_64)
-- | --
Revision | 5b7b76419d50f583022568b6764b630f6ddc9208-refs/branch-heads/5112@{#1309}
OS | macOS Version 12.5 (Build 21G72)
```

Could not reproduce with STR from description.
